ros/api/v1/suggested_instance_types.py

```diff
@@ -0,0 +1,75 @@
+from flask import request
+from sqlalchemy import func
+from flask_restful import Resource, fields, marshal_with
+from ros.api.common.add_group_filter import group_filtered_query
+from ros.lib.utils import (
+    system_ids_by_org_id,
+    org_id_from_identity_header,
+)
+
+from ros.lib.models import (
+    db,
+    PerformanceProfile,
+)
+
+from ros.api.common.pagination import (
+    limit_value,
+    offset_value,
+    build_paginated_system_list_response
+)
+from ros.api.common.instance_descriptions import descriptions
```

Comment:
```suggestion
from ros.api.common.instance_types_helper import descriptions
```

Details are for instance types and not for actual instances so in my opinion it would be good to rename file accordingly.